PurchasesTests/Mocks/MockReceiptFetcher.swift

```diff
@@ -3,22 +3,32 @@
 // Copyright (c) 2020 Purchases. All rights reserved.
 //
 
+@testable import PurchasesCoreSwift
+
 class MockReceiptFetcher: ReceiptFetcher {
     var receiptDataCalled = false
     var shouldReturnReceipt = true
     var shouldReturnZeroBytesReceipt = false
     var receiptDataTimesCalled = 0
+    var receiptDataReceivedRefreshPolicy: ReceiptRefreshPolicy?
+
+    convenience init(requestFetcher: StoreKitRequestFetcher) {
+        self.init(requestFetcher: requestFetcher, bundle: .main)
+    }
 
-    override func receiptData() -> Data? {
+    @objc override public func receiptData(refreshPolicy: ReceiptRefreshPolicy,
+                                           completion: @escaping ((Data?) -> Void)) {
```

Comment:
updated this bit because `receiptData` is no longer the public method for the class